src/audio/speech_to_text/stt_calculator.cc

```diff
@@ -49,6 +49,23 @@ namespace mediapipe {
 
 const std::string STT_SESSION_SIDE_PACKET_TAG = "STT_NODE_RESOURCES";
 
+enum Endpoint{
+    TRANSCRIPTIONS,
+    TRANSLATIONS,
+    UNSUPPORTED
+};
+
+Endpoint getEndpoint(std::string url){
```

Comment:
```suggestion
Endpoint getEndpoint(const std::string& url){
```
dont copy if dont need